module/base/basic_module.cpp

```diff
@@ -55,32 +62,15 @@ void BasicModule::Loop() {
           move(res),
           move(from_machine_id));
     }
-  } else {
-    if (NeedWakeUp()) {
-      HandlePeriodicWakeUp();
-    }
   }
 
-  if (NeedWakeUp()) {
-    auto now = Clock::now();
-    while (now >= wake_up_deadline_) {
-      wake_up_deadline_ += milliseconds(wake_up_every_ms_);
+  // Message from one of the custom sockets
+  for (size_t i = 1; i < poll_items_.size(); i++) {
```

Comment:
This is kind of a confusing block. What is the difference between poll_items_[0] and the rest?

Reply:
`poll_items_[0]` corresponds to the channel socket which goes through the broker. The others are custom sockets and do not go through the broker. I'll add some comment.